docs/getting_started.md

```diff
@@ -328,6 +328,61 @@ See these resources for more information on automated cell type annotation:
 - [Assigning cell types with `SingleR`](https://bioconductor.org/books/release/SingleRBook/)
 - [Cell type assignment chapter in Orchestrating Single Cell Analysis](https://bioconductor.org/books/3.17/OSCA.advanced/cell-cycle-assignment.html)
 
+
+## Working with a merged ScPCA object
+
+Merged ScPCA objects contain all information present in `processed` objects for a given ScPCA project.
+For more information on how these objects were prepared, see {ref}`the section on merged object preparation<processing_information:merged objects`.
+
+Merged object can facilitate joint analysis on multiple samples at the same time, for example to perform [differential expression analysis](https://bioconductor.org/books/3.16/OSCA.multisample/multi-sample-comparisons.html) or integration (i.e., batch correction).
+**However, please be aware that data in merged objects has not been integrated/batch-corrected.**
+If you are interested in performing integration, we recommend consulting with the following resources:
```

Comment:
I think I would expand on this and seprately mention things you can do with merged objects vs. integration. So I would first talk about why you would want to use a merged object - differential expression analysis or any analysis where you are comparing gene level metrics between samples. I would even provide some links to example analyses. 

Then, I would mention that there are other types of analysis that require integration and that we do not perform integration. I would specifically mention that anything where you are comparing cell level metrics (clusters) may need to account for batch correction.

Reply:
>  I would even provide some links to example analyses.

Are you thinking, for example, actual papers which have performed the analyses? Or more general resources about those sorts of analyses?

Reply:
Took a first stab here, let me know what you think! Heading in the right direction?
https://github.com/AlexsLemonade/scpca-docs/pull/230/commits/1d45d8026a8e0630d07500458eae10dbcd85b1ba